keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -24,14 +24,164 @@
 from keras_nlp.tokenizers.word_piece_tokenizer import WordPieceTokenizer
 
 
+@keras.utils.register_keras_serializable(package="keras_nlp")
+class BertTokenizer(WordPieceTokenizer):
+    """A BERT tokenizer using WordPiece subword segmentation.
+
+    This tokenizer class will tokenize raw strings into integer sequences, and
+    is based on `keras_nlp.tokenizers.WordPieceTokenizer`. Unlike the
+    underlying tokenizer, it will check for all special tokens needed by BERT
+    models, and provides a `from_preset()` method to automatically download
+    a matching vocabulary for a BERT preset.
+
+    This tokenizer will not do any truncation or padding of inputs by default.
+    It can be used with a `keras_nlp.models.BertPreprocessor` layer for common
+    input packing, or used directly with a standalone step to combine sequences
+    with any truncation, padding, and special token packing desired.
+
+    If input is a batch of strings (rank > 0), the layer will output a
+    `tf.RaggedTensor` where the last dimension of the output is ragged.
+
+    If input is a scalar string (rank == 0), the layer will output a dense
+    `tf.Tensor` with static shape `[None]`.
+
+    Args:
+        vocabulary: A list of strings or a string filename path. If
+            passing a list, each element of the list should be a single word
+            piece token string. If passing a filename, the file should be a
+            plain text file containing a single word piece token per line.
+        lowercase: If true, the input text will be first lowered before
+            tokenization.
+
+    Examples:
+
```

Comment:
Remove extra line(s)

Reply:
removed one space, leaving one because leaving a break between examples and a further (non-code) heading seemed to be the norm in other docstrings.